### ISSUE TITLE
topgrade panics after update to 10.0.0

### ISSUE THREAD

**Issue by markusgod** [topgrade-rs/topgrade#74]
## What did you expect to happen?
Topgrade performs upgrade sucsessfuly

## What actually happened?
Topgrade panics during upgrade

## Additional Details
- Which operating system or Linux distribution are you using? Opensuse tumbleweed
- How did you install Topgrade? cargo
- Which version are you running? 10.0.0

<details>
❯ topgrade -v
 2022-10-26T08:03:56.400Z DEBUG topgrade::config > Configuration at /home/mark/.config/topgrade.toml
 2022-10-26T08:03:56.400Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: Some({}), post_commands: None, commands: Some({}), git_repos: None, predefined_git_repos: None, disable: None, ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, display_time: None, assume_yes: None, yay_arguments: None, aura_aur_arguments: None, aura_pacman_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, skip_notify: None, bashit_branch: None, only: None, composer: Some(Composer { self_update: None }), brew: Some(Brew { greedy_cask: None, autoremove: None }), linux: Some(Linux { yay_arguments: None, aura_aur_arguments: None, aura_pacman_arguments: None, arch_package_manager: None, show_arch_news: None, trizen_arguments: None, pikaur_arguments: None, pamac_arguments: None, dnf_arguments: None, apt_arguments: None, enable_tlmgr: None, redhat_distro_sync: None, rpm_ostree: None, emerge_sync_flags: None, emerge_update_flags: None }), git: Some(Git { max_concurrency: None, arguments: None, repos: None, pull_predefined: None }), windows: Some(Windows { accept_all_updates: None, self_rename: None, open_remotes_in_new_terminal: None, enable_winget: None }), npm: Some(NPM { use_sudo: None }), yarn: None, vim: None, firmware: Some(Firmware { upgrade: None }), vagrant: None, flatpak: Some(Flatpak { use_sudo: Some(true) }) }
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Cannot find "notify-send"
 2022-10-26T08:03:56.400Z DEBUG topgrade         > Version: 10.0.0
 2022-10-26T08:03:56.400Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2022-10-26T08:03:56.400Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2022-10-26T08:03:56.400Z DEBUG topgrade         > Binary path: Ok("/home/mark/.cargo/bin/topgrade")
 2022-10-26T08:03:56.400Z DEBUG topgrade         > Self Update: false
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Cannot find "doas"
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2022-10-26T08:03:56.400Z DEBUG topgrade::utils  > Cannot find "powershell"
 2022-10-26T08:03:56.400Z DEBUG topgrade::runner > Step "System update"

―― 11:03:56 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2022-10-26T08:03:56.400Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "zypper" "refresh"
Repository 'Visual Studio Code' is up to date.                                                                                                                                   
Repository 'nVidia Graphics Drivers' is up to date.                                                                                                                              
Repository 'Main Repository (NON-OSS)' is up to date.                                                                                                                            
Repository 'Main Repository (OSS)' is up to date.                                                                                                                                
Repository 'Main Update Repository' is up to date.                                                                                                                               
Repository 'Packman Repository' is up to date.                                                                                                                                   
Repository 'Rancher stable packages (rpm)' is up to date.                                                                                                                        
All repositories have been refreshed.
 2022-10-26T08:03:57.531Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "zypper" "dist-upgrade"
Loading repository data...
Reading installed packages...
Warning: You are about to do a distribution upgrade with all enabled repositories. Make sure these repositories are compatible before you continue. See 'man zypper' for more information about this command.
Computing distribution upgrade...
Nothing to do.
 2022-10-26T08:03:58.615Z DEBUG topgrade::runner   > Step "config-update"
 2022-10-26T08:03:58.615Z DEBUG topgrade::runner   > Step "Brew"
 2022-10-26T08:03:58.616Z DEBUG topgrade::utils    > Detected "/home/linuxbrew/.linuxbrew/bin/brew" as "brew"

―― 11:03:58 - Brew ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2022-10-26T08:03:58.616Z DEBUG topgrade::executor > Running "brew" "update"
Already up-to-date.
 2022-10-26T08:04:00.587Z DEBUG topgrade::executor > Running "brew" "upgrade" "--ignore-pinned" "--formula"
 2022-10-26T08:04:01.898Z DEBUG topgrade::runner   > Step "yadm"
 2022-10-26T08:04:01.898Z DEBUG topgrade::runner   > Step "nix"
 2022-10-26T08:04:01.898Z DEBUG topgrade::runner   > Step "guix"
 2022-10-26T08:04:01.898Z DEBUG topgrade::runner   > Step "home-manager"
 2022-10-26T08:04:01.898Z DEBUG topgrade::runner   > Step "asdf"
 2022-10-26T08:04:01.899Z DEBUG topgrade::utils    > Detected "/home/mark/.asdf/bin/asdf" as "asdf"

―― 11:04:01 - asdf ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2022-10-26T08:04:01.899Z DEBUG topgrade::executor > Running "/home/mark/.asdf/bin/asdf" "update"
HEAD is now at 7e7a1fa chore: release 0.10.2 (#1233)
Updated asdf to release v0.10.2
 2022-10-26T08:04:02.566Z DEBUG topgrade::executor > Running "/home/mark/.asdf/bin/asdf" "plugin" "update" "--all"
Updating golang to master
Already on 'master'
Your branch is up to date with 'origin/master'.
Updating nodejs to master
Already on 'master'
Your branch is up to date with 'origin/master'.
Updating java to master
Already on 'master'
Your branch is up to date with 'origin/master'.
 2022-10-26T08:04:03.744Z DEBUG topgrade::runner   > Step "pkgin"
 2022-10-26T08:04:03.744Z DEBUG topgrade::runner   > Step "bun"
 2022-10-26T08:04:03.744Z DEBUG topgrade::utils    > Path "/home/mark/.config/emacs" doesn't exist
 2022-10-26T08:04:03.744Z DEBUG topgrade::utils    > Path "/home/mark/.emacs.d" doesn't exist
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.doom.d does not exists
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.vim does not exists
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.config/nvim does not exists
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.ideavimrc does not exists
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.intellimacs does not exists
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > /home/mark/.zshrc is a file. Checking /home/mark
 2022-10-26T08:04:03.744Z DEBUG topgrade::steps::git > Checking if /home/mark is a git repository
 2022-10-26T08:04:03.745Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(32768)), stdout: "", stderr: "fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n" }
 2022-10-26T08:04:03.745Z DEBUG topgrade::steps::git > /home/mark/.tmux does not exists
 2022-10-26T08:04:03.745Z DEBUG topgrade::steps::git > Checking if /home/mark/.config/fish is a git repository
 2022-10-26T08:04:03.747Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(32768)), stdout: "", stderr: "fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n" }
 2022-10-26T08:04:03.747Z DEBUG topgrade::steps::git > /home/mark/.config/openbox does not exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::steps::git > /home/mark/.config/bspwm does not exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::steps::git > /home/mark/.config/i3 does not exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::steps::git > /home/mark/.config/sway does not exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "Git repositories"
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zr"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "antibody"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "antigen"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zgenom"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zplug"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zinit"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zi"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2022-10-26T08:04:03.747Z DEBUG topgrade::runner     > Step "zim"
 2022-10-26T08:04:03.747Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.750Z TRACE topgrade::executor   > Output of "zsh" "-c" "[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}": Output { status: ExitStatus(unix_wait_status(256)), stdout: "", stderr: "" }
 2022-10-26T08:04:03.750Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2022-10-26T08:04:03.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2022-10-26T08:04:03.750Z DEBUG topgrade::utils      > Path "/home/mark/.oh-my-zsh" exists

―― 11:04:03 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2022-10-26T08:04:03.753Z TRACE topgrade::executor   > Output of "zsh" "-c" "test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM": Output { status: ExitStatus(unix_wait_status(256)), stdout: "", stderr: "" }
 2022-10-26T08:04:03.753Z DEBUG topgrade::steps::zsh > Running zsh returned exit status: 1: . Using default path: /home/mark/.oh-my-zsh/custom
 2022-10-26T08:04:03.753Z DEBUG topgrade::steps::zsh > oh-my-zsh custom dir: /home/mark/.oh-my-zsh/custom
 2022-10-26T08:04:03.754Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom is a git repository
 2022-10-26T08:04:03.755Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.755Z DEBUG topgrade::steps::git > /home/mark/.oh-my-zsh/custom/example.zsh is a file. Checking /home/mark/.oh-my-zsh/custom
 2022-10-26T08:04:03.755Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom is a git repository
 2022-10-26T08:04:03.756Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.756Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins is a git repository
 2022-10-26T08:04:03.757Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.757Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/example is a git repository
 2022-10-26T08:04:03.759Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.759Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-completions is a git repository
 2022-10-26T08:04:03.760Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-completions\n", stderr: "" }
 2022-10-26T08:04:03.760Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting is a git repository
 2022-10-26T08:04:03.762Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting\n", stderr: "" }
 2022-10-26T08:04:03.762Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions is a git repository
 2022-10-26T08:04:03.763Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions\n", stderr: "" }
 2022-10-26T08:04:03.763Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/nerdctl is a git repository
 2022-10-26T08:04:03.764Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/nerdctl\n", stderr: "" }
 2022-10-26T08:04:03.764Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/telepresence is a git repository
 2022-10-26T08:04:03.765Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/telepresence\n", stderr: "" }
 2022-10-26T08:04:03.765Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/epinio is a git repository
 2022-10-26T08:04:03.766Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/epinio\n", stderr: "" }
 2022-10-26T08:04:03.766Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes is a git repository
 2022-10-26T08:04:03.767Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.767Z DEBUG topgrade::steps::git > /home/mark/.oh-my-zsh/custom/themes/example.zsh-theme is a file. Checking /home/mark/.oh-my-zsh/custom/themes
 2022-10-26T08:04:03.767Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes is a git repository
 2022-10-26T08:04:03.769Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2022-10-26T08:04:03.769Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes/powerlevel10k is a git repository
 2022-10-26T08:04:03.770Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(unix_wait_status(0)), stdout: "/home/mark/.oh-my-zsh/custom/themes/powerlevel10k\n", stderr: "" }
Pulling custom plugins and themes
 2022-10-26T08:04:03.771Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.772Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.773Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.774Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.775Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.777Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.778Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(unix_wait_status(0)), stdout: "origin\n", stderr: "" }
 2022-10-26T08:04:03.780Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "655576f336f4c956b59a5f4424c5ffd361964b35\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-completions
 2022-10-26T08:04:03.782Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "122dc464392302114556b53ec01a1390c54f739f\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
 2022-10-26T08:04:03.784Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "dd46501b9c838ad7424569f98c0194fc0e808bed\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/telepresence
 2022-10-26T08:04:03.785Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "81adf993a1baeaea503fa9d187f97ef6b86e763a\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/epinio
 2022-10-26T08:04:03.786Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "a411ef3e0992d4839f0732ebeb9823024afaaaa8\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2022-10-26T08:04:03.787Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "6f153508a932e25cca2b914ffcddb73eb2f98de9\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/nerdctl
 2022-10-26T08:04:03.789Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "8091c8a3a8a845c70046684235a01cd500075def\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/themes/powerlevel10k
 2022-10-26T08:04:04.268Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "122dc464392302114556b53ec01a1390c54f739f\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
 2022-10-26T08:04:04.271Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "6f153508a932e25cca2b914ffcddb73eb2f98de9\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/nerdctl
 2022-10-26T08:04:04.284Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "655576f336f4c956b59a5f4424c5ffd361964b35\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-completions
 2022-10-26T08:04:04.294Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "a411ef3e0992d4839f0732ebeb9823024afaaaa8\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2022-10-26T08:04:04.324Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "8091c8a3a8a845c70046684235a01cd500075def\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/themes/powerlevel10k
 2022-10-26T08:04:05.425Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "dd46501b9c838ad7424569f98c0194fc0e808bed\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/telepresence
 2022-10-26T08:04:05.497Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(unix_wait_status(0)), stdout: "81adf993a1baeaea503fa9d187f97ef6b86e763a\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/epinio
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/topgrade-10.0.0/src/steps/git.rs:234:15

<pre>

</pre>
</details>

**Comment by markusgod** [comment]
Looks like https://github.com/topgrade-rs/topgrade/blob/b565aa5f71a850a8d302b16d31cf60379508ff64/src/steps/git.rs#L233-L234 is causing problems.
If there are no error, find returns None, and None unwraps and panic.
Wrapping error in something like 
```rust
        if let Some(err) = error {
            err?;
        }
```
Probably should be enough.

**Comment by Osveron** [comment]
Based on logs seems related to #73. @markusgod can you try if #71 fixes it for you?

**Comment by markusgod** [comment]
Yep, that's another way, it works, thanks.

**Comment by DottoDev** [comment]
Closed by #71
